Library/Formula/khard.rb

```diff
@@ -0,0 +1,44 @@
+class Khard < Formula
+  desc "Console carddav client."
+  homepage "https://github.com/scheibler/khard"
+  url "https://github.com/scheibler/khard/archive/v0.4.0.tar.gz"
+  sha256 "4140fb06a1da63a5eb26edc7d3e3c01b99c464276a9e590b57ee2451c4416f77"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  resource "vobject" do
+    url "https://pypi.python.org/packages/source/v/vobject/vobject-0.8.1c.tar.gz"
+    sha256 "594113117f2017ed837c8f3ce727616f9053baa5a5463a7420c8249b8fc556f5"
+  end
+  resource "configobj" do
```

Comment:
Please add spacing between the resources. It's hard to read quickly bunched together.